clang/docs/ReleaseNotes.rst

```diff
@@ -464,6 +464,8 @@ Improvements to Clang's diagnostics
 
 - Clang now diagnoses ``[[deprecated]]`` attribute usage on local variables (#GH90073).
 
+- Clang now diagnoses misused reasoned ``delete("reason")`` warnings only in pedantic mode. (#GH109311).
```

Comment:
```suggestion
- Clang now diagnoses ``= delete("reason")`` extension warnings only in pedantic mode rather than on by default. (#GH109311).
```

Reply:
@AaronBallman The CI checks have passed, if no further reviews are required, could you please merge it? thanks

Reply:
At this point, I'd prefer if you'd obtain commit privileges and land the changes yourself -- you have a number of patches under your belt, and it's better for folks to land their own changes in case of needing reverts, etc.

If there's a reason you don't want/can't get commit rights, feel free to let me know privately or otherwise, and I can certainly land the changes for you.

Reply:
@AaronBallman Got it, thanks.